editors_checklist.md

```diff
@@ -0,0 +1,51 @@
+# Editors' checklist
+
+Upon submission:
+
+-   Check that template has been properly filled out
+-   Tag issue appropriately with `package`, `topic:`, etc. tags
+-   Check against policies for [fit](policies.md#fit) and [overlap](policies.md#fit)
+    If reject, close issue, direct authors to other groups/repos as appropriate.
+-   Check that mandatory parts of template are complete.  If not, close issue,
+    direct authors toward appropriate instructions.
+-   Assign reviewers.  Use the #onboarding slack channel for discussion about
+    potential reviewers.  Assign reviewer by tagging in the issue with the
+    following format to activate  the automated reminders bot:
+   
+      Reviewers: @githubname 
+      Due date: YYYY-MM-DD
+
+During review:
+
+-   Check in with reviewers and authors occaisionally.
+
+After review:
+
+-   If authors intend to submit to CRAN, check against CRAN gotchas.
+-   If packages work with an unstable API or are expected to have significant
+    additions post-review, they should be migrated to `ropenscilabs`. Otherwise,
+    they should me be migrated to `ropensci`.
+-   Add rOpenSci footer to README.  
+-   Close issue.
+-   Check if authors are interested in writing summary blog post for blog.
+    Otherwise, write a short summary for regular "round-up" blog posts.
```

Comment:
Okay, makes sense, yeah a monthly blog post would be good - Any objections from @ropensci/leadership ?  

Right, task view maintainers